kotlinx-coroutines-test/common/src/TestBuilders.kt

```diff
@@ -196,6 +196,7 @@ internal const val DEFAULT_DISPATCH_TIMEOUT_MS = 60_000L
 internal suspend fun <T: AbstractCoroutine<Unit>> runTestCoroutine(
```

Comment:
Do we still need generic parameter here instead of `AbstractCoroutine<Unit>`?

Reply:
There's still the `TestScope`/`TestCoroutineScope` difference.